nekoyume/Assets/_Scripts/Extension/ItemExtensions.cs

```diff
@@ -57,5 +57,10 @@ public static string GetCPText(this Costume costume, CostumeStatSheet sheet)
 
         public static bool TryGetOptionInfo(this ItemUsable itemUsable, out ItemOptionInfo itemOptionInfo) =>
             ItemOptionHelper.TryGet(itemUsable, out itemOptionInfo);
+
+        public static bool HasElementType(this ItemType type)
+        {
+            return type != ItemType.Consumable && type != ItemType.Material;
+        }
```

Comment:
How about below? I like to check positively.😁
```suggestion
        public static bool HasElementType(this ItemType type) => type == ItemType.Costume ||
                                                                                                      type == ItemType.Equipment;
```